composer.json

```diff
@@ -12,13 +12,13 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": ">=5.4.0",
         "symfony/framework-bundle": "~2.3|~3.0",
         "symfony/console": "~2.1|~3.0",
         "symfony/form": "~2.1|~3.0",
         "symfony/property-access": "~2.3|~3.0",
-        "ruflin/elastica": "~2.1",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "ruflin/elastica": "3.2.*"
```

Comment:
Because elastica is not semver yet
On May 20, 2016 06:55, "Martin Hasoň" <notifications@github.com> wrote:

> In composer.json
> <https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1069#discussion_r63991188>
> :
>
> >          "symfony/framework-bundle": "~2.3|~3.0",
> >          "symfony/console": "~2.1|~3.0",
> >          "symfony/form": "~2.1|~3.0",
> >          "symfony/property-access": "~2.3|~3.0",
> > -        "ruflin/elastica": "~2.1",
> > -        "psr/log": "~1.0"
> > +        "psr/log": "~1.0",
> > +        "ruflin/elastica": "3.2.*"
>
> Why not ^3.0?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1069/files/e8f2dad53e04b230386f0bce5b6ec939e06c0d18#r63991188>
>